build/tasks/clean.js

```diff
@@ -12,8 +12,9 @@ var del = require("del");
  */
 gulp.task("clean", function () {
 	return del([
+		"aot-compiled/*",
 		"dist/*",
-		"tmp/"
+		"tmp/*"
```

Comment:
There isn't. Actually let me clean up the clean task to delete all the generated folders themselves.